charts/ratify/templates/provider.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: externaldata.gatekeeper.sh/v1alpha1
 kind: Provider
 metadata:
-  name: ratify-provider
+  name: ratify-verification-provider
```

Comment:
Is this provider already part of a release? If yes, isn't changing the name now a breaking change that'll result in 2 providers `ratify-provider` and `ratify-verification-provider` when a user upgrades using these charts?

Reply:
thanks for pointing this out. yeah this will definitely be a breaking change. I changed the name to be more consistent with the addition of the mutation provider but I think it's more detrimental to do this. I will revert.